libs/components/modals/src/lib/modules/confirm/confirm.service.ts

```diff
@@ -11,8 +11,7 @@ import { SkyConfirmModalContext } from './confirm-modal-context';
 import { SkyConfirmComponent } from './confirm.component';
 
 /**
- *  The `SkyConfirmService` provider launches the confirmation dialog.
- * Within the service, you can specify the dialog's message and customize the button text.
+ * Launches a confirmation dialog.
```

Comment:
```suggestion
 * Launches a dialog.
```
We delete "confirmation" elsewhere, so should we delete it here too?